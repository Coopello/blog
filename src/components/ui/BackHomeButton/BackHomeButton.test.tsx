import renderer from "react-test-renderer";

import { BackHomeButton } from "./";

describe("ui/BackHomeButton", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<BackHomeButton />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
