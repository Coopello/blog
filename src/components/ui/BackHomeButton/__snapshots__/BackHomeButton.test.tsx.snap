// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/BackHomeButton Snap Shot 1`] = `
<button
  className="rounded hover:opacity-60 bg-custom-white h-9 w-[256px] undefined"
  onClick={[Function]}
>
  トップページに戻る
</button>
`;
